client/src/components/Header/ImageGallery.jsx

```diff
@@ -17,7 +17,7 @@ export default function HeaderGallery() {
       sx={{ width: "100vw", height: "450" }}
```

Comment:
General comment, Reacutier page the image gallery is offset a good but left and a little lower than other pages.